Gemfile

```diff
@@ -33,24 +33,35 @@ else
   gem 'sqlite3'
 end
 
-framework, *version = ENV.fetch('FRAMEWORK', 'rails').split('-')
-version = version.join('-')
-
-case version
-when 'master'
-  gem framework, github: "#{framework}/#{framework}"
-when /.+/
-  gem framework, "~> #{version}.0"
-else
-  gem framework
+## Install Framework
+
+GITHUB_REPOS = { 'grape' => 'ruby-grape/grape'}
+
+frameworks = ENV.fetch('FRAMEWORK', 'rails').split(',')
+frameworks_versions = frameworks.inject({}) do |frameworks, str|
+  framework, *version = str.split('-')
+  frameworks.merge(framework => version.join('-'))
 end
 
-gem 'activerecord-jdbcsqlite3-adapter', platform: :jruby
+frameworks_versions.each do |framework, version|
+  case version
+  when 'master'
+    gem framework, github: (GITHUB_REPOS[framework] || "#{framework}/#{framework}")
+  when /.+/
+    gem framework, "~> #{version}.0"
+  else
+    gem framework
+  end
+end
 
-unless version =~ /^(master|6)/
-  gem 'delayed_job', require: nil
+if frameworks_versions.key?('rails')
+  unless frameworks_versions['rails'] =~ /^(master|6)/
+    gem 'delayed_job', require: nil
+  end
 end
 
+gem 'activerecord-jdbcsqlite3-adapter', platform: :jruby
+
```

Comment:
@mikker Is this ok with you? Now we can test two frameworks (ex: `FRAMEWORK=rails-6.0,grape-1.2`)

Reply:
I like it a lot!